### PR TITLE
Footer: editorial changes

### PR DESCRIPTION
**Pull request by Garneauma**
Changes included in this PR: 

- Editorial change - Update code sample for footer version 2 English - The code sample in the "version 2.0 (deprecated)" should match the version 2 of the working example.
- Editorial change - Update code sample for footer version 1 French - The code sample in the "version 1.0 (déprécié)" should match the version 1 of the working example.
- Editorial change - Change the parameter noFooterAbout for the "no-footer-about-en.html" to be replace by "no-footer-about-gov-en.html". (English and French) This to remove the confusion between the h2 heading (About this site) vs only the h3 heading (Government of Canada).
- Editorial change - Change the parameter noFooterSite in "no-footer-site-en.html" for something like "no-footer-support-links-en.html" (English and French)
- Editorial change - Change the other working example which show a combination of the renamed "noFooterSite" and "noFooterAbout".
- Editorial change - The code sample of "only-footer-contextual-en.html" and "only-footer-contextual-fr.html" don't match the self test page. Probably some page property are missing here. Structurally the code match.
- Editorial change - Document the change to the skip-link component. In the footer documentation page and in the skip-link documentation page.
- Rename the page property noFooterAbout for noFooterAboutGov.
- Rename the page property noFooterSite for noFooterSupportLinks.
- Move the new i18nText variable from the core into the overwritable i18n.

Changes related to WET-289

### CODE REVIEW

**Comment by duboisp** [comment]
Pre-approved upon successful testing and code review

**Review by duboisp — DISMISSED** [review]
Approved after the merge conflict is resolved.

For the next version we know that we will need to change the reference to "support" for something else because it now refer to the first section of the site footer. We should receive some guidance soon. But here, it's only about editorial change for the version 12 and will set a baseline for the next major version of the site footer which is imminent.

**Comment by Garneauma** [comment]
> Approved after the merge conflict is resolved.

Rebased and conflicts resolved. Further changes will be applied on PR #2043.

**Review by duboisp — APPROVED** [review]
Completed the code review and local testing. This is now ready to be merged.

This change is only about content change and impact the unversionned gcweb-jekyll

**Pull request merged**
